.prepare_deploy

```diff
@@ -22,29 +22,42 @@ This binary was compiled on $(date -u).
 $(git log -n 3)
 END
 
-for file in ./*.exe
-do
-  zip "${file%.*}.zip" README.txt LICENSE.txt "$file"
-done
-
-for file in *.linux-x86_64
-do
-  base="${file%.*}"
-  cp "$file" "shellcheck"
-  tar -cJf "$base.linux.x86_64.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
-  rm "shellcheck"
-done
-
-for file in *.linux-armv6hf
-do
-  base="${file%.*}"
-  cp "$file" "shellcheck"
-  tar -cJf "$base.linux.armv6hf.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
-  rm "shellcheck"
-done
+if [ "$TRAVIS_OS_NAME" = 'linux' ];
```

Comment:
You can just `shopt -s nullglob` instead of checking the OS to just let it skip any pattern where there are no matching outputs